yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+JSONStream@^1.0.3:
```

Comment:
Ok, yarn is for dependency management right? I'm not sure I understand why is it necessary to include in this PR that just changes text.